CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changes by Version
 -------------------
 
 ### Backend Changes
+* The default value for the flag `query.max-clock-skew-adjustment` has changed to `0s`, meaning that the clock skew adjustment is now disabled by default. See [#1459](https://github.com/jaegertracing/jaeger/issues/1459).
```

Comment:
Should this be listed as a breaking change as well?

Reply:
It does! I swear I thought I was adding it to the *Breaking* changes, not *Backend*...

Reply:
Moved.